fundingmanager.go

```diff
@@ -383,25 +383,39 @@ func (f *fundingManager) Start() error {
 
 		confChan := make(chan *lnwire.ShortChannelID)
 		timeoutChan := make(chan struct{})
-
 		go func(ch *channeldb.OpenChannel) {
 			go f.waitForFundingWithTimeout(ch, confChan, timeoutChan)
```

Comment:
Note that this goroutine will cease waiting for the funding tx to confirm after 288 blocks, regardless of if we delete the channel from the database or not. Instead, in the case we are the initiator, we should launch `waitForFundingConfirmation`

Reply:
Does that not get called within `waitForFundingWithTimeout` ?

Reply:
Yes, but it gets cancelled when the timeout happens. If we are initiator, we want it to never cancel.

Reply:
I've moved the initiator logic into `waitForFundingConfirmation`, so that it will only trigger the timeout if we were not the initiator.